src/components/Rating.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+function Rating(props) {
+  let { rating } = props;
+
+  let result = () => {
+    switch (rating) {
+      case rating  === 0:
+        return '☆☆☆☆☆';
+
+      case rating  === 1:
+        return '★☆☆☆';
+
+      case rating === 2:
+        return '★★☆☆☆';
+
+      case  rating  === 3:
+        return '★★★☆☆';
+
+      case  rating  === 4:
+        return '★★★★☆';
+
+      case  rating  === 5:
+        return '★★★★★';
```

Comment:
This logic is correct but you're checking if the rating is equal to a number, so this will only work if the rating is exactly the same number that you have in your condition.

What you need to do is, instead of checking if the rating is equal (===) you need to check if it is bigger (>=) or less (<=)